SwiftMonkey/Monkey.swift

```diff
@@ -147,12 +147,17 @@ public class Monkey {
         }
     }
 
-    /// Generate random events forever, or until the app crashes.
-    public func monkeyAround() {
-        while true {
+    /// Generate random events or fixed-interval events based forever, for a specific duration or until the app crashes.
+    ///
+    /// - Parameter duration: The duration for which to generate the random events.
+    ///
+    /// - TimeInterval: Set to .infinity by default.
```

Comment:
The parameter name is `duration` so it should be:
> /// - duration: Set to .infinity by default.

**Improvement**: You can surround .infinity with the \`\` code markers to make it look even better in the Quick Help:
> /// - duration: Set to \`.infinity\` by default.

![screen shot 2017-10-12 at 13 47 38](https://user-images.githubusercontent.com/1027098/31494586-3372c5a2-af54-11e7-8e36-ab2e7a39c8ee.png)

**Suggestion**

How about we put the `Set to .infinity by default.` in the main description of the `duration` parameter? It would look like this:

```
/// - Parameter duration: The duration for which to generate the random events.
///                       Set to `.infinity` by default.
```

![screen shot 2017-10-12 at 13 48 59](https://user-images.githubusercontent.com/1027098/31494628-7a43c0e4-af54-11e7-8340-3c2041a87319.png)

What do you think @grycyk ?